frontend/src/Student/pages/Dashboard.js

```diff
@@ -1,17 +1,22 @@
 import React from 'react'
 import { Container, Row, Col } from 'reactstrap'
+import { connect } from 'react-redux'
 
 import Shows from '../containers/Shows'
 
-const Dashboard = () => (
+const Dashboard = props => (
   <Container>
     <Row>
       <Col>
-        <Shows />
+        <Shows studentUsername={props.studentUsername}/>
```

Comment:
When there are no past/current shows we should probably add a simple message like "No shows are currently running, come back later"

Reply:
Done :)